multigeiger/userdefines-example.h

```diff
@@ -63,3 +63,20 @@
 // 0x2A38: Heart Rate Sensor Position --> sends TUBE_TYPE
 // 0x2A39: Heart Rate Control Point --> allows to reset "energy expenditure", as required by service definition
 #define SEND2BLE false
+
+// Play an alarm sound when radiation level is too high?
+// Activates when either accumulated dose rate reaches the set threshold (s. below)
+// or when the current dose rate is higher than the accumulated dose rate by the set factor (s. below)
+// ! Requires a valid tube type to be set in order to calculate dose rate
```

Comment:
True, but rather should be noted for 76 and 82.

Reply:
Added it in all three description blocks.